docs/getting-started/concepts.md

```diff
@@ -14,8 +14,8 @@ With Concrete ML, you can train a model on clear or encrypted data, then deploy
 
 1. **Quantization:** Quantization converts inputs, model weights, and all intermediate values of the inference computation to integer equivalents. More information is available [here](../explanations/quantization.md). Concrete ML performs this step in two ways depending on model type:
 
-   - During training (Quantization Aware Training)
-   - After training (Post-training Quantization)
+   - During training (Quantization Aware Training): Add explanation
+   - After training (Post Training Quantization): Add explanation
```

Comment:
Can we add some simple explanations here about QAT and PTQ in the "concepts" doc?
In other docs whenever there's a mention of QAT and PTQ, I link here as a reference.